packages/dart/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: parse_server_sdk
 description: Dart plugin for Parse Server, (https://parseplatform.org), (https://back4app.com)
-version: 4.0.2
+version: 5.0.0
```

Comment:
Go directly to version 5? Isn't it better to have a beta version?

Reply:
Yes, we can do that.

@mtrezza  what do you think? should we publish this as beta?
Like this `5.0.0-beta.1`

[publishing-prereleases](https://dart.dev/tools/pub/publishing#publishing-prereleases)

Reply:
We do not have this repository set up to be publish beta releases. If we publish beta releases on the same branch as stable releases we may run into trouble when adding auto-release later on.